README.md

```diff
@@ -20,15 +20,15 @@ Let us know if you want to get involved.
 
 The plugin is designed to support Swift 5 syntax.
 
-| Feature             | Swift                                                        | Objective-C                                                   |
-|---------------------|--------------------------------------------------------------|---------------------------------------------------------------|
-| Size                | YES                                                          | IN PROGRESS                                                   |
-| Issues              | [SwiftLint 0.47.0](https://github.com/realm/SwiftLint) rules | [OCLint 22.02](https://oclint.org/) rules                     |
-| Tests               | YES                                                          | IN PROGRESS                                                   |
-| Coverage            | YES                                                          | YES                                                           |
-| Complexity          | IN PROGRESS                                                  | IN PROGRESS                                                   |
-| Syntax highlighting | IN PROGRESS                                                  | IN PROGRESS                                                   |
-| Security            | [mobsfscan 0.10.0](https://github.com/MobSF/mobsfscan) rules | [mobsfscan 0.10.0](https://github.com/MobSF/mobsfscan) rules  |
+| Feature             | Swift                                                        | Objective-C                                                  |
+|---------------------|--------------------------------------------------------------|--------------------------------------------------------------|
+| Size                | YES                                                          | YES                                                          |
+| Issues              | [SwiftLint 0.47.0](https://github.com/realm/SwiftLint) rules | [OCLint 22.02](https://oclint.org/) rules                    |
+| Tests               | YES                                                          | YES                                                          |
```

Comment:
Why tests is also added here?

Reply:
Because I did it last week, but forgot to update the table :)